sdk/cognitivelanguage/Azure.AI.Language.Conversations/tests/Generated/Models/AnalyzeConversationJobState.cs

```diff
@@ -22,14 +23,8 @@ public partial class AnalyzeConversationJobState : JobState
         /// <exception cref="ArgumentNullException"> <paramref name="jobId"/> or <paramref name="tasks"/> is null. </exception>
         internal AnalyzeConversationJobState(DateTimeOffset createdDateTime, string jobId, DateTimeOffset lastUpdatedDateTime, State status, ConversationTasksStateTasks tasks) : base(createdDateTime, jobId, lastUpdatedDateTime, status)
         {
-            if (jobId == null)
-            {
-                throw new ArgumentNullException(nameof(jobId));
-            }
-            if (tasks == null)
-            {
-                throw new ArgumentNullException(nameof(tasks));
-            }
+            Argument.AssertNotNull(jobId, nameof(jobId));
```

Comment:
While I'm glad to see the helpers I added years back used, it *could* cause an ever so slight regression depending on the jitter. I wonder if we should add `[MethodImpl(MethodImplOptions.AggressiveInlining)]` to the `Assert*` and `Check*` methods. @KrzysztofCwalina, what do you think?

This shouldn't hold up the PR, though. I can open a separate issue and it would just be Core update anyway.

Reply:
Opened #32071 to track.